egs/callhome_diarization/v1/local/VB_resegmentation.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
```

Comment:
This script should probably go in diarization/ rather than local/ because we'll want to be able to use it from other recipes as well.